src/components/TeacherViews/Onboarding/Onboarding.view.tsx

```diff
@@ -69,8 +75,23 @@ export const OnboardingView = () => {
   };
 
   const onFormSubmit = async (data: OnboardingFormProps) => {
-    // TODO: something with this data
-    console.log("onboarding form values: ", data);
+    try {
+      // TODO: something with this data
+      console.log("onboarding form values: ", data);
+
+      await onboardUser();
```

Comment:
We can post extra data here to mark the user as a teacher. For now this action simply flips the user to the onboarded state in Auth0.